src/main/java/frc/robot/Robot.java

```diff
@@ -74,6 +74,8 @@ public void robotPeriodic() {
 
         enabledTrigger.update(isEnabled());
 
+        RobotState.getInstance().update();
```

Comment:
Why is that needed? this is not done for any other system. Perhaps another way is possible (hint: commands)